vhost/src/backend.rs

```diff
@@ -254,6 +254,9 @@ pub trait VhostBackend: std::marker::Sized {
     /// This is no longer used.
     fn reset_owner(&self) -> Result<()>;
 
+    /// Disable all rings and reset the internal device state.
+    fn reset_device(&self) -> Result<()>;
```

Comment:
IIUC this is supported only by vhost-user, so what about adding it just in `VhostUserFrontend` ?

Reply:
That makes sense, yes. Will do.